flow/benchmarks/figureeight0.py

```diff
@@ -66,7 +67,7 @@
     # scenario's documentation or ADDITIONAL_NET_PARAMS component)
     net=NetParams(
         no_internal_links=False,
-        additional_params=ADDITIONAL_NET_PARAMS,
+        additional_params=deepcopy(ADDITIONAL_NET_PARAMS),
```

Comment:
nit: does `ADDITIONAL_NET_PARAMS.copy()` not work here? u used it once above, and from my experience that works fine too